boards/riscv/beaglev_starlight_jh7100/doc/index.rst

```diff
@@ -41,7 +41,7 @@ Flashing
    any other serial terminal that supports xmodem based file transfers.
 
 #. BeagleV Starlight uses uart for flashing. Refer to `BeagleV Getting Started
-   <https://wiki.seeedstudio.com/BeagleV-Getting-Started/>`_
+   <https://web.archive.org/web/20210625011721/https://wiki.seeedstudio.com/BeagleV-Getting-Started/>`_
```

Comment:
agreed. btw, isn't this board obsolete as well? https://www.hackster.io/news/the-beaglev-starlight-board-is-canceled-but-starfive-partners-with-radxa-for-a-replacement-design-454a55982dd0

Reply:
While the board is still in the tree, at least providing a link to the archive seemed like a better option than to completely remove it.

Reply:
just saying because you removed links for other obsolete boards

Reply:
If you are talking about `msp_exp432p401r_launchxl`, I removed them because they were links to PDF files and I assumed `archive.org` only archives web pages.

But, after checking, it looks like they archive non-web pages too, including PDF. I will update these with the links to the archive.

Reply:
uhh ... looks like this PR got merged before I could update this.

I will leave this as-is since these boards are going to be removed in the near future ...

Reply:
would you mind opening an issue to track boards to be deleted?